src/main/scala/nl/knaw/dans/easy/stage/EasyStageDataset.scala

```diff
@@ -107,8 +106,8 @@ object EasyStageDataset {
       _ <- EasyStageFileItem.createFileSdo(sdoDir, relativePath, "objectSDO" -> parentSDO
       )(FileItemSettings(
         sdoSetDir = Some(s.sdoSetDir),
-        pathInStorage = Some(new File(relativePath)),
-        storageBaseUrl = s.bagStorageLocation,
+        dsLocation = None,
```

Comment:
I am using the parameter name that [Fedora uses](https://wiki.duraspace.org/display/FEDORA38/REST+API#RESTAPI-modifyDatastream). Granted, some familiarity with the Fedora API is assumed.